src/main/java/frc/robot/commands/DefaultDrive.java

```diff
@@ -19,11 +19,10 @@
 /** An example command that uses an example subsystem. */
 public class DefaultDrive extends Command {
 	
-	SlewRateLimiter xVelocityFilter = new SlewRateLimiter(slewRateLimit);
-	SlewRateLimiter yVelocityFilter = new SlewRateLimiter(slewRateLimit);
-	SlewRateLimiter rotationalVelocityFilter = new SlewRateLimiter(slewRateLimit);
+	SlewRateLimiter xVelocityFilter = new SlewRateLimiter(driveSlewRateLimit);
```

Comment:
Please use `SwerveConstants.driveSlewRateLimit`
Dont use .* that is a bad practice on Java